tests/std/tests/GH_002431_byte_range_find_with_unreachable_sentinel/test.cpp

```diff
@@ -4,6 +4,7 @@
 #include <algorithm>
 #include <cassert>
 #include <cstddef>
+#include <isa_availability.h>

```

Comment:
why must we include this?

Reply:
Presumably because this appears below:

https://github.com/microsoft/STL/blob/c8d1efb6d504f6392acf8f6d01fd703f7c8826c0/tests/std/tests/GH_002431_byte_range_find_with_unreachable_sentinel/test.cpp#L30-L36